spec/grape/dsl/routing_spec.rb

```diff
@@ -82,9 +82,17 @@ class Dummy
           subject.route(:any)
         end
 
-        it 'defines a new endpoint' do
-          expect { subject.route(:any) }
-            .to change{ subject.endpoints.count }.from(0).to(1)
+        context 'defines new endpoint' do
+          it 'defines a new endpoint' do
+            expect { subject.route(:any) }
+              .to change{ subject.endpoints.count }.from(0).to(1)
+          end
+
+          it 'uniq the duplicating endpoint' do
```

Comment:
You don't need to put these two tests together, just add a new test at the same level that says 'does not duplicate identical endpoints'.